constraints.md

```diff
@@ -219,16 +210,19 @@ For example, if you create a foreign key on `orders.customer` that references `c
 - Each value inserted or updated in `orders.customer` must exactly match a value in `customers.id`.
 - Values in `customers.id` that are referenced by `orders.customer` cannot be deleted or updated. However, `customers.id` values that _aren't_ present in `orders.customer` can be.
 
+{{site.data.alerts.callout_success}}If you plan to use Foreign Keys in your schema, consider using <a href="interleave-in-parent.html">interleaved tables</a>, which can dramatically improve statement's performance.{{site.data.alerts.end}}
```

Comment:
"statement's performance" to "statements' performance" or "statement performance".